src/dealDashboard/dealDiscussion/discussionThread/discussionThread.html

```diff
@@ -49,15 +49,15 @@ <h3 class="subtext">
             index="${$index}"
             replies-to.bind="comment.replyTo ? (threadDictionary[comment.replyTo] ? threadDictionary[comment.replyTo] : deletedComment) : ''"
             replies-to-profile.bind="comment.replyTo ? (threadProfiles[threadDictionary[comment.replyTo].author]) : ''"
-            is-authorized.bind="deal.isUserRepresentativeOrLead"
+            is-authorized.bind="authorized"
```

Comment:
would be nice to have a more meaningful name than "authorized".  Authorized to do what?